libs/coin-tester/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  speculos:
+    # Speculos isn't supported for Mac M1 yet. You have to build the image locally as described in the README.
+    # Otherwise you can use the official image
+    # image: ghcr.io/ledgerhq/speculos:latest
+    image: speculos:latest
```

Comment:
Do we have a way to condition this based on a value in env instead ? Because the CI won't be able use the container this way 🤔